src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -132,6 +128,8 @@ export const ScheduleItemNG: React.FC<ScheduleItemNGProps> = ({
     [userId, event]
   );
 
+  const [isEventLinkCopied, setIsEventLinkCopied] = useState(false);
```

Comment:
Maybe a good idea to have this `useState` before the other code that uses `isEventLinkCopied` and `setIsEventLinkCopied`. Can also go as top line since it is short and depends on no other code.

Reply:
moved https://github.com/sparkletown/sparkle/pull/2183/commits/2be26c4e66c5595df4c893f818cc364b297a21b3